packages/Sandblocks-Core/SBResultsView.class.st

```diff
@@ -0,0 +1,119 @@
+Class {
```

Comment:
this class, as well as the Exploriants classes, probably belong into Sandblocks-Babylonian, not -Core, right?